HED-schema-reduced.mediawiki

```diff
@@ -0,0 +1,495 @@
+HED version: v1.4.0-restruct
+
+'''Syntax'''  
+
+HED tags can only be separated by commas. Use semicolons (;) instead of commas in event descriptions since otherwise extra commas could confuse HED parsers.  
+From version 2.2, HED adheres to http://semver.org/ versioning. 
+
+!# start hed 
+
+'''Event''' 
+* Category <nowiki>{required, requireChild, position=1} [This is meant to designate the reason this event was recorded. Every event should have at least one category, but may have multiple categories.] </nowiki> 
+** Sensory presentation <nowiki> [Something that can be perceived.]</nowiki>
+*** Experimental <nowiki>{extensionAllowed} [Primary category for time-locked stimulus events.]</nowiki>
+*** Environmental <nowiki>{extensionAllowed}[An environmental event, for example, a change in experimental context such as walking on dirt versus sidewalk or a loud noise in the next room.]</nowiki> 
+** Participant action <nowiki>[Any action engaged in by the participant.]</nowiki>
+*** Task-related {extensionAllowed} <nowiki>[An action taken by a participant in response to an experimental event.]</nowiki>
+*** Incidental {extensionAllowed} <nowiki>[A non-task-related action taken by the participant -- for example yawning, standing up during a seated experiment, or responding to a loud noise unrelated to the experiment.]</nowiki>
+** Data feature <nowiki>{extensionAllowed} [An event derived from the data itself, possibly inserted in a post-hoc fashion.]</nowiki>
+** Experiment 
+*** Control <nowiki>{extensionAllowed} [Information about states and events that control the experiment.]</nowiki>
```

Comment:
Thinking about the removed Experiment/Control/Activity category, should we change the description of Experiment/Control category to something like: "An event pertaining to setup and/or control the experiment"?